src/components/ButtonPanel.js

```diff
@@ -1,40 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
+  const { clickHandler } = props;
+
   return (
     <div className="btn-panel">
       <div className="row">
-        <Button symbol="AC" />
-        <Button symbol="+/-" />
-        <Button symbol="%" />
-        <Button symbol="÷" />
+        <Button symbol="AC" clickHandler={clickHandler} />
+        <Button symbol="+/-" clickHandler={clickHandler} />
+        <Button symbol="%" clickHandler={clickHandler} />
+        <Button symbol="÷" clickHandler={clickHandler} />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

Reply:
Thank you, this would definitely DRY the code. I'll add this as a feature request and tackle it some other time. Good suggestion!